python/src/opendp/mean.py

```diff
@@ -0,0 +1,67 @@
+from opendp.trans import *
+from opendp.meas import *
+
+
+class DPMean(object):
```

Comment:
What if instead of making a class, you wrote a function `make_dp_mean` that returns a measurement that obeys the given epsilon? Then you could still chain it with combinators (amplifiers, compositors) down the line. I feel like you'd also be better off if you pulled make_split_dataframe outside, because you probably won't want to parse and split the dataframe for every statistic.

```python
splitter = make_split_dataframe(col_names=col_names)
dp_mean = make_dp_mean(index, constant, lower, upper, n, epsilon)
(splitter >> dp_mean)(data)

# down the line when you have multiple measurements:
dp_sum = make_dp_sum(index, constant, lower, upper, n, epsilon2)
comp = make_sequential_composition_static_distances(1, [(dp_mean, epsilon), (dp_sum, epsilon2)])
mean, sum = (splitter >> comp)(data)
```

I also recommend exposing the input distance in your interface as well.

Reply:
This is a great idea, thanks